src/mlpack/core/optimizers/adam/adam.hpp

```diff
@@ -43,11 +45,15 @@ namespace optimization {
  *   year    = {2014},
  *   url     = {http://arxiv.org/abs/1412.6980}
  * }
+ * @article{
+ *   title   = {On the convergence of Adam and beyond},
+ *   url     = {https://openreview.net/pdf?id=ryQu7f-RZ}
+ * }
```

Comment:
We can at least add a year here if you like (2018 I guess), and then update this reference when the paper is published.